### PR TITLE
Remove unused interval parameter from spinners structure; Fix code style

### PR DESCRIPTION
**Pull request by progsmile**
Optimized structure of spinner map - just leave frames only inside
Add `.editorconfig` file
Fix code style

---
This change is [Reviewable](https://reviewable.io/reviews/blackode/elixir_cli_spinners/3)

### CODE REVIEW

File: lib/cli_spinners.ex [hunk 1]
```diff
@@ -176,11 +176,8 @@ defmodule CliSpinners do
   However you can break that by pressing the `Ctrl + C` for two times.
   """
   def demo do
-    functions = CliSpinners.Spinners.__info__(:functions)
-    for {fun_name, _} <- functions do
-      fun_name |> Atom.to_string() |> IO.write
```
Comment (progsmile) [inline lib/cli_spinners.ex:181]: @blackode Do we actually need this line?

**Comment by blackode** [comment]
@progsmile Great Work. That was awesome... 👏

**Comment by progsmile** [comment]
@blackode Thanks)

File: lib/cli_spinners.ex (hunk 1 above)
Comment (blackode) [reply lib/cli_spinners.ex:181]: I think so. It justs give the hint about the `definition` it is running at present. So, the user might understand what animations is going on...

Comment (progsmile) [reply lib/cli_spinners.ex:181]: The matter is that it doesn't give definition. You may checkout to earlier commits and ensure, so, leave it as it is :)

**Pull request merged**